source/server/hot_restart_impl.cc

```diff
@@ -108,17 +108,22 @@ HotRestartImpl::HotRestartImpl(Options& options)
 
 Stats::RawStatData* HotRestartImpl::alloc(const std::string& name) {
   // Try to find the existing slot in shared memory, otherwise allocate a new one.
+  Stats::RawStatData* unused = nullptr;
   std::unique_lock<Thread::BasicLockable> lock(stat_lock_);
   for (Stats::RawStatData& data : shmem_.stats_slots_) {
     if (!data.initialized()) {
```

Comment:
Ugh, I put it back to the broken state when I was cleaning up the 2-->1 loops stuff, but I reversed the order so the test passes.

Reply:
Oh wait, now I've confused myself, I think this is still ok..  Either way, need a better test, which I'll write right now.